src/engine/strat_engine/pool.rs

```diff
@@ -163,9 +163,25 @@ impl StratPool {
 
 impl Pool for StratPool {
     fn create_filesystems<'a, 'b>(&'a mut self,
-                                  _specs: &[&'b str])
+                                  specs: &[&'b str])
                                   -> EngineResult<Vec<(&'b str, Uuid)>> {
-        unimplemented!()
+        let names = BTreeSet::from_iter(specs);
+        for name in names.iter() {
```

Comment:
I get a compiler error if I make that change.   The error is related to:

 result = Vec::new(). (and the return of the vec)

the trait bound `str: std::marker::Sized` is not satisfied [E0277]:
`str` does not have a constant size known at compile-time
tuple elements must have `Sized` type
the trait `std::marker::Sized` is not implemented for `str`